### ISSUE TITLE
TWE: Create Template Issue: Rapid Qualitative Analysis (RQA): Pre-process

### ISSUE THREAD

**Issue by pandanista** [hackforla/internship#618]
### Overview
We need to create a template issue for TWE: Research: Rapid Qualitative Analysis (RQA): Pre-process (named in the format as `TWE: Role: Direct Object: Action Verb`)] so that we have a consistent, transparent, defined, and self-supporting structure for people creating them.

#### Customize this issue
 - Customize this issue by adding the Name of Issue Template. 
    - _Note: An issue template's name is different from the issue title created from the template._
    - [x] Search for `[Replace NAME OF ISSUE TEMPLATE]` by using the browser search (**Mac**: ⌘+F; **PC**: Ctrl+F)
    - [x] Replace the keyword string (including text and the brackets) below with the name of your issue template according to the template naming convention: `TWE: Role: Direct object: Action verb`
        - The role is the primary owner of the issue template (e.g., Research, Design, Product)
        - Direct object should be the item that we're updating or working on (e.g., Wiki, Interviews, Roadmap)
        - Action verb should be the main descriptor of the action items (e.g., Update, Create, Transcribe)

#### Customize the issue settings (located in right sidebar)
Change the issue settings to ensure the issue is properly categorized and easy to manage.
- [x] Under Assignees, assign yourself.
- [x] Under Projects, choose gear, choose Repository, and choose TWE: Project Board.
- [x] Add the milestone: Project Management
- [x] Add a feature label specific to what the template is related to, if available.  If not clear what you should use, ask product.

#### Customize template draft
Create a draft of new template, which will be known as our "draft".
- [x] Copy resource 1.01 Content to copy for draft to a new comment on this issue.
- [x] In your draft, fill out the top part (called front matter) with the following information.  See resource 1.02 Front Matter Resources section below for definitions and examples.
    - [x] Name
    - [x] About
    - [x] Title
    - [x] Labels
- [x] Define any Dependencies (or remove the dependency section if there is no dependency)
- [x] Customize the Overview section
- [x] Fill out the Action Items section (in your draft) by defining the steps that are required to be done, keeping the following in mind:
   - [x] The person working on this issue might be new to the org, so the issue should be able to be worked on without any prior knowledge
   - [x] If the person needs to add links, or put documents in specific locations, the place where they can do that action must be linked in the resources section
      - [x] The links in the resources section provided by the issue creator should be in the same order as the Action Items 
      - [x] Refer to resource 1.03 Example Action Items for Finding Wiki Page and Template for an example of how to write clear instructions that avoid compound steps (steps within a step).
   - [x] Number each resource in the order each resource appears in the Action Items
       - [x] Group resources into numbers and decimals (e.g., Resource 1.01, Resource 2.01)
       - [x] Group the resources based on whether they are available when making the issues or they need to be created from this issue
           - If the resource is available, then group it under the **Resources for creating this issue** section (e.g., Resource 1.01, Resource 1.02, Resource 1.03)
           - If the resource will be delivered or created from this issue, then group it under the **Resource links gathered during the making of this issue** section (e.g., Resource 2.01, Resource 2.02, Resource 2.03)
- [x] If documents or folders (sheets, docs, presentations, dashboard, etc.) will be created, find the document template that should be used by checking resource 1.04 WIKI Templates by role pages.  
  - [x] If there is a template
     - [x] Add a link to the template into the ` #### Resources for creating this issue`  section of your draft
  - [ ] If there is no template yet
    - [ ] Put a placeholder into the` #### Resources for creating this issue`  section of the draft `[TWE: Template Issue: [Role]: [Replace ACTION VERB][Replace DIRECT OBJECT]`
- [ ] Get sign-off by Product on the draft

#### If this is for a template draft that needs testing at a later stage, please use the following handoff steps:
- [x] Add `feature: template creation (step 1)` to this issue
- [x] Once the template draft is finished, assign a team member or yourself to create testing issues 
- [x] Once the testings issues are completed, get sign-off by Product on the template draft
- [x] Create or update the template issue on the repo 
- [x] Add the link to the new issue template once created to resource 2.01.  Find by clicking on the **Get started** button to the right of the new template listed on [new issue page](https://github.com/hackforla/internship/issues/new/choose)
- [x] Update the appropriate place in the wiki for your template (see resources section)

#### Resources for creating this issue
- 1.01 Content to copy for draft
```
Draft of template issue for `TWE: Research: Rapid Qualitative Analysis (RQA): Pre-process`
---
**name:** TWE: Template Issue: [Replace ROLE]: [Replace ACTION VERB][Replace DIRECT OBJECT]
**about:** This issue helps you structure your template issue to be consistent with our other template issues in [Replace DIRECT OBJECT]
** title:** TWE: [Replace ROLE]: [Replace DIRECT OBJECT]: [Replace ACTION VERB]
**labels:** 'feature: board / google drive / tools, feature: missing, milestone: missing, role: missing, size: missing', 'Participant Type: missing', 'complexity: missing'
**assignees:**

---
#### Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- Customize the issue
    - [ ] Copy and paste entire template text in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
    - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
    - [ ] Search for `[Replace`
    - [ ] Replace with whatever the keyword or information is asking for.
- Customize the issue settings (located in right sidebar)
    - [ ] Under Labels, add labels for any labels identified as missing
    - [ ] Remove the corresponding missing labels
    - [ ] Under Projects, choose gear, choose "Repository," and choose TWE: Project Board.
    - [ ] Add the milestone: [Replace MILESTONE]
    - [ ] Add a feature label specific to what the template is related to, if available. If not clear what you should use, ask product.
- Customize dependencies 
    - [ ] If no dependencies, delete dependency section
    - [ ] If dependencies, add the dependencies (any issue or reason that would prevent the action items from being completed)
- Remove this section.
    - [ ] Delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
We need to [Replace ACTION VERB] [Replace DIRECT OBJECT] for [Replace REASON].

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, coding, etc.

### Resources
#### Resources for creating this issue
- 1.01
- 1.02

#### Resources gathered during the completion of this issue
- 2.01
- 2.02
```

- 1.02 Front Matter Resources
   <details>
   <summary>Front Matter Fields and Meaning</summary>

   - name: (name that appear on the new issue tab)
   - about: (is the description that appears next to the issue template on the new issue tab)
   - title: (the title that appears when the issue is created.  User then customizes this title for their issue)
       - Use the naming convention: TWE: Role: Direct object: Action verb
       - If more than one word should go into the direct object spot, use multiple square brackets with the keyword (see example below)
   - labels: (choose the labels that will be the same each time, and choose missing labels for the ones that depend on content)
   - assignees:  
   </details>
   <details>
   <summary>Front Matter Field Example</summary>

   - name: 'TWE: Template Issue: Research: Create Roadmap'
   - about:  This issue helps you structure your template issue to be consistent with our other template issues in roadmap
   - title: 'TWE: Research: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TYPE OF RESEARCH METHOD]: Roadmap : Create'
   - labels: 'feature: research', 'feature: roadmap', 'milestone: missing', 'role: UI/UX research', 'Participant Type: missing', 'Research 000: missing', 'complexity: missing'
   - 'size: 0.50pt'
   - assignees: ''
   </details>

- 1.03 Example Action Items for Finding Wiki Page and Template
   <details>
   <summary>Example Action Items</summary>

   ```
   ### Action Items
   - Customize wiki page
        - [ ] Go to link 1.01 Internship Wiki, Join the team page
        - [ ] Find wiki page for this issue by looking at role pages.
        - [ ] Copy the link for the wiki page.
        - [ ] Add the link to Resource 2.01
           - [ ] Add the name of the link in square brackets
           - [ ] Add the URL in parenthesis without a space between the square bracket and parenthesis
   - Customize document template.
       - [ ] Go to link 1.02 Design Process Overview
       - [ ] Find document template that should be used in this issue.
       - [ ] Copy the link for the template.
       - [ ] Add the link to Resource 2.02
           - [ ] Add the name of the link in square brackets
           - [ ] Add the URL in parenthesis without a space between the square bracket and parenthesis
   ```

   </details>

   <details>
   <summary>Example Resource Lists</summary>

   ```
   ### Resources for creating this issue
   - 1.01. [Internship Wiki, Join the team page](https://github.com/hackforla/internship/wiki/Joining-the-Team)
   - 1.02. [Design Process Overview](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

   ### Resource links gathered during the making of this issue
   - 2.01. [Wiki page for [Replace TYPE OF ROLE]]
   - 2.02. [TWE: Template: [Replace NAME OF TEMPLATE]]
   ```

   </details>

- 1.04 WIKI Templates by role pages
   - [TWE: WIKI: UX Research Guides Templates](https://github.com/hackforla/internship/wiki/UX-Research-Guides-Templates) 
   - [Design](https://github.com/hackforla/internship/wiki/Design-Process-Overview)
   - [Product](https://github.com/hackforla/internship/wiki/Product)
   - [Business Analyst / Data Science]
   - [Software Engineer]
   - [Marketing]
   - default document templates, in case you don't have one yet, can be found on the [Design Process Overview page](https://github.com/hackforla/internship/wiki/Design-Process-Overview)

#### Resources gathered during the completion of this issue
- 2.01 [TWE: IS[Replace YY]: RP[Replace 000]: Rapid Qualitative Analysis (RQA): Pre-process](https://github.com/hackforla/internship/issues/new?assignees=&labels=complexity%3A+medium%2C+Feature%3A+research+analysis%2C+feature%3A+research%2C+milestone%3A+missing%2C+Participant+Type%3A+missing%2C+Research+000%3A+missing%2C+role%3A+UI%2FUX+research%2C+size%3A+1pt&projects=&template=research--rapid-qualitative-analysis--rqa---pre-process-.md&title=TWE%3A+IS%5BReplace+YY%5D%3A+RP%5BReplace+000%5D%3A+Rapid+Qualitative+Analysis+%28RQA%29%3A+Pre-process)

**Comment by pandanista** [comment]
Draft of template issue for `[Replace NAME OF ISSUE TEMPLATE]`
---
**name:** TWE: Template Issue: [Replace ROLE]: [Replace ACTION VERB][Replace DIRECT OBJECT]
**about:** This issue helps you structure your template issue to be consistent with our other template issues in [Replace DIRECT OBJECT]
** title:** TWE: [Replace ROLE]: [Replace DIRECT OBJECT]: [Replace ACTION VERB]
**labels:** 'feature: board / google drive / tools, feature: missing, milestone: missing, role: missing, size: missing', 'Participant Type: missing', 'complexity: missing'
**assignees:**

---
#### Instructions for creating this issue
The person who creates this issue should use these links to add links to the resources section and customize this issue
- Customize the issue
    - [ ] Copy and paste entire template text in this issue into text editor (ie., "TextEdit" on _Mac_, "Wordpad" on _PC_)
    - Use Find/Replace (**Mac:** ⌘+F or **PC**: Ctrl+F) to search for and replace the keyword strings (including text and the brackets) below with the appropriate values 
    - [ ] Search for `[Replace`
    - [ ] Replace with whatever the keyword or information is asking for.
- Customize the issue settings (located in right sidebar)
    - [ ] Under Labels, add labels for any labels identified as missing
    - [ ] Remove the corresponding missing labels
    - [ ] Under Projects, choose gear, choose "Repository," and choose TWE: Project Board.
    - [ ] Add the milestone: [Replace MILESTONE]
    - [ ] Add a feature label specific to what the template is related to, if available. If not clear what you should use, ask product.
- Customize dependencies 
    - [ ] If no dependencies, delete dependency section
    - [ ] If dependencies, add the dependencies (any issue or reason that would prevent the action items from being completed)
- Remove this section.
    - [ ] Delete this section when all the checkboxes above are complete (i.e., the issue assignee will never see this part)

### Dependencies

### Overview
We need to [Replace ACTION VERB] [Replace DIRECT OBJECT] for [Replace REASON].

### Action Items
A STEP BY STEP LIST OF ALL THE TASK ITEMS THAT YOU CAN THINK OF NOW EXAMPLES INCLUDE: Research, reporting, coding, etc.

### Resources
#### Resources for creating this issue
- 1.01
- 1.02

#### Resources gathered during the completion of this issue
- 2.01
- 2.02

**Comment by pandanista** [comment]
A draft for the complete RQA process. Save this comment for the later phases.

### Dependencies
- RP001's research plan needs to be finalized. See #27 
- RP001's transcript has been transcribed, de-identified and cleaned up. See #355, #476, #478, #479, #480, #481
- RP001's research method needs to be researched and step-by-step instructions need to be provided. See #338 

### Overview
We need to re-analyze the data collected for RP001 to generate research findings, synthesize insights, and provide product recommendations.

### Action Items

#### Customize Resource Links
- Customize Resource for Wiki Page Link
    - [ ] Go to the wiki page: Research Output Overview (Resources # 1.01)
    - [ ] Choose the link in the Research by Plan Number section
    - [ ] Locate relevant wiki page for RP001
    - [ ] Copy the link for the wiki page
    - [ ] In Resources # 2.01, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked <img width="100" alt="Screenshot 2023-02-21 at 6 47 15 PM" src="https://user-images.githubusercontent.com/57029070/220508425-5d0c0a1b-ef87-4ef2-a683-d4a21d5b2672.png">

- Customize Resource for this Research Plan's Google Drive Folder
    - [ ] Open the Google Drive's Research by Type Folder (Resources # 1.02)
    - [ ] Choose the `Intern` folder
    - [ ] Choose the `RP001` folder
    - [ ] Copy the link of the `RP001` folder
        1. Choose the three vertical dots on the right side of the `RP001` folder  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
        1. Choose "Share"
        1. Choose "Get link"
        1. Choose "Copy Link" and "Done"
    - [ ] In Resources # 2.02, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
    - [ ] Open the `RP001` folder in Google Drive (Resources # 2.02)
    - [ ] Locate the research plan in `RP001` folder  
        - [ ] Choose RP001's research plan in Google Doc
        - [ ] Copy the link of the `RP001` research plan
            1. Choose the three vertical dots on the right side of the `RP001` research plan  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
            1. Choose "Share"
            1. Choose "Copy Link" 
            1. The "Link copied" notification pops up
    - [ ] In Resources # 2.03, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
    - [ ] Locate the RP001's de-identified transcript(s) in `RP001` folder (Resources # 2.02)
        - [ ] Choose RP001's de-identified transcript(s)
        - [ ] Copy the link of the `RP001` de-identified transcript(s)
            1. Choose the three vertical dots on the right side of the `RP001` de-identified transcript(s)  <img width="50" alt="Screenshot 2023-10-10 at 5 58 13 PM" src="https://github.com/hackforla/internship/assets/57029070/f8b81823-6e66-40c1-b885-881c356baa96">
            1. Choose "Share"
            1. Choose "Copy Link" 
            1. The "Link copied" notification pops up
    - [ ] In Resources # 2.04, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked

#### Review Resources to Get Familiar with the Hamilton's Process of Rapid Qualitative Analysis (RQA)
- [ ] If you are new to RQA, review the list of resources to get familiar with the RQA process and techniques (Resources 1.03 - 1.05)
- [ ] If you are experienced with RQA, you may skip reviewing the documents or review them for a refresher

#### Create the Rapid Qualitative Analysis Domain Names
- [ ] Read through the research plan to get familiar with the study (Resources 2.03 and 2.04)
    - [ ] Focus on the interview protocol/interview guide in the research plan Google doc
- Create a neutral domain name spreadsheet that corresponds with each interview question (an example refers to p. 22 in the notes of Hamilton's Process/Approach in Resources 1.03)
    - [ ] Go to `TWE: Research: Rapid Qualitative Analysis Domain Names Template` linked in Resources 1.06  
    - [ ] Choose `Use Template`
    - [ ] Copy the following texts
        ```
        TWE: IS21: RP001: Intern Exit Interviews: Rapid Qualitative Analysis Domain Names
        ```
    - [ ] Paste the texts you copied into the formula bar of the spreadsheet to replace the default title (see the highlighted part in the screenshot)  <img width="500" alt="Screenshot 2024-08-29 at 2 01 19 PM" src="https://github.com/user-attachments/assets/d3a9d66d-f816-4bad-875c-e68fd6d6b94b">
    - [ ] Choose File > Rename
    - [ ] Paste the texts you copied into the text box to replace the default name of the file
    - [ ] Move the file to RP001's folder in the shared Google Drive
        1. Choose File > Move > All Locations > Shared drives > Internship > Internships > Research > Research by Plan # and Name > RP001 > Move
        1. There will be a question to confirm change the ownership of the documents, and choose Move
    - [ ] Copy the link of the Rapid Qualitative Analysis Domain Names file of RP001
        1. Choose Share
        1. Choose Copy link and Done
    - [ ] In Resources # 2.05, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
- Fill out a neutral domain name spreadsheet that corresponds with each interview question (an example refers to p. 22 in the notes of Hamilton's Process/Approach in Resources 1.03)
    - [ ] Go to the research plan doc (linked in Resources 2.03) to locate the interview protocol/questions
    - [ ] Copy and paste each interview question into a row in the Interview Question column (column A) in the Rapid Qualitative Analysis Domain Names file (linked in Resources 2.05 if it is not open already)
    - [ ] Draft the domain based on the interview question in the Domain column (column B) in the Rapid Qualitative Analysis Domain Names file

#### Peer Review Stage of the Domain Names
- [ ] Notify the Research Lead that the domain names are finished by leaving a comment in this issue
- [ ] Add `needs peer review` label
- [ ] Research Lead assigns a team member to conduct a peer review
- [ ] The peer reviewer conducts the review by leaving notes in the Peer Review column (column C) in the Rapid Qualitative Analysis Domain Names file (linked in Resources 2.05)
- [ ] The peer reviewer notifies the Research Lead and the authors of the domain names that the peer review is completed
- [ ] The authors of the domain names discuss with the peer reviewer and research lead on the feedback and incorporates the feedback if needed
- [ ] The authors of the domain names will add `Ready for Research Lead` label once the domain names are finalized, and move the issue to `Question/Review` column
- [ ] The authors of the domain names will update the team about the changes made 
- [ ] Research lead will review the updated domain names

**_Phase 2 here or after this section?_**
#### Create the Transcript Summary Template
- Draft the summary template (an example refers to p. 23 in the notes of Hamilton's Process/Approach in Resources 1.03)
    - [ ] Include basic information about the interview at the top
    - [ ] Include the neutral domains created above
    - [ ] Include a row called "Other Observations, for materials that doesn’t fit into any domain" at the end of the spreadsheet
    - [ ] Include space for important quotations at the end
- Test the summary template
    - [ ] Test the summary template on a subset of transcripts - **_do we need to more specific on how many transcripts to include as a subset?_**
    - [ ] Assess the quality of the neutral domains created
    - [ ] Compare the summarizing "styles" across the team
    - [ ] Measure the time of completion and control for optimal summary length
- [ ] Revise the template until consistency has been established
- [ ] [Optional] Create an instructional handout for transcript summaries and distribute it to the team

#### Conduct Transcript Summaries
- [ ] Distribute transcripts across the team. The division of transcripts should be customized to suit research needs.
- [ ] Complete the summary for each transcript (an example refers to p. 27 in the notes of Hamilton's Process/Approach in Resources 1.03)
- [ ] Check the quality of the summaries /maybe a peer review step by step guide here

#### Conduct Matrix Analysis
- [ ] Create a matrix
    - The matrix format should be customized to suit research needs. 
    - The most commonly used format is respondent * domain (an example refers to p. 32 in the notes of Hamilton's Process/Approach in Resources 1.03)
- [ ] Transfer the summaries into the matrix
- [ ] Analyze the Matrix. The objective and method of matrix analysis should be customized to suit research needs. Common options are:
    - [ ] Develop a codebook and conduct code analysis
    - [ ] Identify themes and conduct thematic analysis
    - [ ] Create memos to inform future data collection
    - [ ] Evaluate the data collection quality (if the matrix is in the format of interviewee * domain)

#### Prepare reports/presentations

**_Need to edit the rest of the draft for RP001's data analysis_**
- [ ] Create a case study report using the template with HfLA branding (Resources # 1.11)
    - [x] Choose `Use Template`
    - Update the title of the case study 
        - [x] Copy the following texts
        ```
        TWE: IS22: RP017: Intern Activity Dashboard Case Study Report
        ```
        - [x] Paste the texts you copied into the title page to replace the default title <img width="400" alt="Screenshot 2024-05-28 at 5 11 21 PM" src="https://github.com/hackforla/internship/assets/57029070/8dee370d-e458-41c1-844d-5cee47c0f1a3">
        - [x] Choose File > Rename
        - [x] Paste the texts you copied into the text box to replace the default title <img width="400" alt="Screenshot 2024-05-28 at 5 10 26 PM" src="https://github.com/hackforla/internship/assets/57029070/42bc304e-5631-4b2c-9872-d561d9cc2db9">
    - [x] Move the file to RP017's folder in the shared Google Drive
        1. Choose File > Move > All Locations > Shared drives > Internship > Internships > Research > Research by Plan # and Name > RP017 > Move
        1. There will be a question to confirm change the ownership of the documents, and choose Move
    - [x] Copy the link of the case study report of RP017 
        1. Choose Share
        1. Choose Copy link and Done
    - [ ] In Resources # 2.05, place the link you just copied between parentheses at the end of the line with no space in between the right bracket `]` and the left parenthesis `(`, so it turns into a hyperlink
    - [ ] Choose "Update comment" in Github and make sure all the checkboxes above have been checked
- Complete each section of a case study report
    - [x] Background
        - Provide a short summary about the significance of the research, rationale for conducting the research, and aims
    - [x] Objective
        - Discuss case study method and rationale for use, participant recruitment, protocol/development of interview guide
    - [x] Study Design/Data Collection
        - Discuss case study method and rationale for use, participant recruitment, protocol/development of interview guide 
    - [x] Data Analysis
        - Outline your data analysis plan, note themes of interest for case study 
    - [ ] Key Findings
        - Provide your examination of the case, discuss findings as they relate to themes focused on during interview, present relevant facts and key problems identified
    - [ ] Recommendations
        - Present possible solutions/changes that are realistic, noting possible pros/cons
        - Support solution with outside evidence
        - Discuss any specific strategies for accomplishing proposed solutions/changes
    - [ ] Review each section
    - [ ] Update the Table of Content (page 2) of the case study report Google doc to reflect the page and section changes 


### Peer review and finalization stages we might need for later phases

#### Peer review stage of the transcript summaries
- [ ] Notify the Research Lead that the transcript summaries are finished by leaving a comment in this issue
- [ ] Add `needs peer review` label
- [ ] Research Lead assigns a team member to conduct a peer review
- [ ] The peer reviewer conducts the review by leaving comments in the transcript summaries
- [ ] The peer reviewer notifies the Research Lead and the authors of the transcript summaries that the peer review is completed
- [ ] The authors of the transcript summaries discuss with the peer reviewer and research lead on the feedback and incorporates if needed
- [ ] The authors of the transcript summaries will add `Ready for Research Lead` label once the transcript summaries are finalized and move the issue to `Question/Review` column
- [ ] The authors of the transcript summaries will update the team about the changes made 
- [ ] Research lead will review the transcript summaries

#### Peer review stage of the matrix analysis
- [ ] Notify the Research Lead that the matrix analysis is finished by leaving a comment in this issue
- [ ] Add `needs peer review` label
- [ ] Research Lead assigns a team member to conduct a peer review
- [ ] The peer reviewer conducts the review by leaving comments in the matrix analysis
- [ ] The peer reviewer notifies the Research Lead and the authors of the matrix analysis that the peer review is completed
- [ ] The authors of the matrix analysis discuss with the peer reviewer and research lead on the feedback and incorporates if needed
- [ ] The authors of the matrix analysis will add `Ready for Research Lead` label once the matrix analysis is finalized and move the issue to `Question/Review` column
- [ ] The authors of the matrix analysis will update the team about the changes made 
- [ ] Research lead will review the matrix analysis

#### Finalize the matrix analysis based on peer review feedback
- [ ] After discussion on the feedback, the authors of the matrix analysis will address any necessary changes if there is any remaining.
- [ ] The authors of the matrix analysis will notify the Research Lead that the matrix analysis is ready to be reviewed by product by leaving a comment in this issue
- [ ] Research lead will add `Ready for Product` label once ready for product review
- [ ] Product Team reviews and provides feedback 
- [ ] Product Team notifies the Research Team about any changes needed
- [ ] Research Team incorporates any necessary feedback
- [ ] Product sign-off

#### Update wiki page and action item spreadsheet
- [ ] Update relevant WIKI page (See resources)
- [ ] Create the action item spreadsheet for this research plan
- [ ] Update the action item spreadsheet based on research findings and insights
- [ ] Share the action item spreadsheet with Product
- [ ] Product sign-off

### Resources/Instructions

#### Resources needed to complete issue
1.01 [Wiki Intern Research Overview Page](https://github.com/hackforla/internship/wiki/Intern-Research-Overview)
1.02 [Research by Participant Type Folder on Google Drive](https://drive.google.com/drive/folders/1f5Qgq-ikT_UwcgRBuoBamqY0Wacsg9f5?usp=drive_link)
1.03 [Qualitative Methods in Rapid Turn-Around Health Services Research](https://www.hsrd.research.va.gov/for_researchers/cyber_seminars/archives/video_archive.cfm?SessionID=780) **MUST READ**
1.04 [Linda's Presentation to UXR Team on How to Conduct Rapid Qualitative Analysis](https://docs.google.com/presentation/d/12j99CF8fnvdb7WOWxDn1_CO_UODjoL23HXZUqujbveA/edit?usp=sharing)
1.05 [a step-by-step checklist for conducting a Rapid Qualitative Analysis](https://github.com/hackforla/internship/issues/338#issuecomment-2192408993) - This issue comment provides more context on the checklist this issue is following 
1.06 [TWE: Research: Rapid Qualitative Analysis Domain Names Template](https://docs.google.com/spreadsheets/d/1O8nU0iKQiecZzkuyGP-uJoqVwIwlFZNSBnUAAKIy6-U/template/preview)
1.07 [TWE: Template: IS[Replace YY]: RP[Replace 000]: Rapid Qualitative Analysis Transcript Summaries](https://docs.google.com/spreadsheets/d/1kHnMpIK6AHEoxFY9XhGMb5JRWPBnuDjZzahG2vovGJQ/template/preview)


~1.11 [TWE: Research: Case Study Template](https://docs.google.com/document/d/13GBl5vmR6eeB69XgtM-vz2hvwbS--L27DbSbQ7BrLx0/template/preview)~

#### Resources gathered during the completion of the issue
2.01 [RP001 Wiki Page]
2.02 [Google Drive RP001 Folder]
2.03 [RP001 Research Plan]
2.04 [RP001 De-identified Transcript]
2.05 [TWE: IS21: RP001: Intern Exit Interviews: Rapid Qualitative Analysis Domain Names]
~2.05 [RP017 Case Study Report]~

**Comment by pandanista** [comment]
@jinyan0425: As a follow up to our template testing meeting on 9/3/2024,
- The template draft for RQA pre-process is in [this comment](https://github.com/hackforla/internship/issues/618#issuecomment-2327661203).
- Please test the template by doing [step 2: set up a test](https://github.com/hackforla/internship/issues/new?assignees=&labels=feature%3A+missing%2C+feature%3A+research%2C+feature%3A+test+a+template+%28step+2a%29%2C+good+first+issue%2C+milestone%3A+missing%2C+Participant+Type%3A+TWE+Team+Member%2C+role%3A+UI%2FUX+research%2C+size%3A+3pt%2C+time+sensitive&projects=&template=research--test--issue-template.md&title=Research%3A+Test%3A+Issue+Template%3A+%5BReplace+with+NAME+OF+TEMPLATE%5D) and step 3: creating a testing issue (#616)
- I moved the original draft draft I was showing to you in [this comment](https://github.com/hackforla/internship/issues/618#issuecomment-2327713858)
- Let me know if you have any questions. Thank you!

**Comment by pandanista** [comment]
The dependencies might need some rephrasing.

Here is an example from research plan and goals chart template, providing instructions on how to identify if there is a dependency issue.
- [ ] If a research plan is missing, 
        - [ ] Choose the gear in the Labels section <img width="500" alt="Screenshot 2023-04-19 at 8 06 27 PM" src="https://user-images.githubusercontent.com/57029070/233248009-99d8d10c-112a-4bc9-8fd5-b1e0098aea23.png">
        - [ ] Put on the `dependency` label
        - [ ] Copy
        ```
        ### Dependencies
        - [ ] The research plan(s) needs to be created. See issue # 
        ```
        - [ ] Choose "Edit" under the three dots to access the GitHub Write mode
        - [ ] Paste what you just copied to the beginning of the issue above the `Overview` section
        - [ ] Choose `Update comment` <img width="100" alt="Screenshot 2023-02-21 at 6 47 15 PM" src="https://user-images.githubusercontent.com/57029070/220508425-5d0c0a1b-ef87-4ef2-a683-d4a21d5b2672.png">
        - [ ] Choose `Issues` located on the top of the GitHub page to open a new tab in your browser <img width="600" alt="Screenshot 2023-06-15 at 5 18 07 PM" src="https://github.com/hackforla/internship/assets/57029070/bcee2deb-8032-4ea3-863f-dd7e5b3bb898">
        - [ ] In the search bar, include the corresponding missing research plan number, i.e. RP001, to do a search to see if the research plan has an existing issue. Make sure to check both open and closed issues. <img width="700" alt="Screenshot 2023-06-15 at 4 18 57 PM" src="https://github.com/hackforla/internship/assets/57029070/7ab3bec1-1fa4-4fc3-a801-362785686d53">
        - [ ] If there is an existing issue for creating that research plan, add the issue number after "See issue # " in the dependencies section so we can track  the status of the research plan issue
            - [ ] Assign the existing research plan issue to yourself if no one is assigned, so you can start working on that since it is a dependency
        - [ ] If the issue for creating that research plan does not exist among the issues, that means we haven't started drafting a research plan yet. Please go ahead and create a new issue by clicking [create a research plan template](https://github.com/hackforla/internship/issues/new?assignees=&labels=complexity%3A+medium%2C+feature%3A+research+plan%2C+milestone%3A+missing%2C+Participant+Type%3A+missing%2C+Research+000%3A+missing%2C+role%3A+UI%2FUX+research%2C+size%3A+2pt&projects=&template=research--create-a-research-plan.md&title=TWE%3A+IS%5BReplace+YY%5D%3A+RP%5BReplace+000%5D%3A+Research+Plan%3A+Create)
            - [ ] Customize the new issue using the missing research plan's number
            - [ ] Once the new issue is created, include the issue number after "See issue # " in the dependencies section above
            - [ ] Assign the new issue to yourself, so you can start working on the new issue since it is a dependency for this issue
            - [ ] Move the new issue to the `In Progress` column on the [TWE: Project Board](https://github.com/orgs/hackforla/projects/64)
        - [ ] Make a comment in this issue as an update to inform the team that there is a missing research plan and include the issue number for creating the research plan
        - [ ] Put this issue back to the icebox         
        - [ ] Remove yourself from this issue
    - [ ] Choose "Update comment" in Github and make sure all the action item you have accompolished above have been checked

**Comment by jinyan0425** [comment]
@pandanista Ying, I see this general example of creating dependencies. I feel the problem with this template is that the dependency is specific to RP1 and a more general guideline may not be applicable.

**Comment by jinyan0425** [comment]
@pandanista Based on yesterday's UX meeting, should I start creating the template for RQA—Step 2: TWE: Create Template Issue: Rapid Qualitative Analysis (RQA): Transcript Summary Creation?

**Comment by pandanista** [comment]
@jinyan0425 Yep, that looks like a good start.

**Comment by jinyan0425** [comment]
@pandanista 

I will draft the action items for creating and testing the transcript summary template in this comment!

**Comment by jinyan0425** [comment]
@pandanista  I drafted the action items for `Create the Transcript Summary Template` in this [comment](https://github.com/hackforla/internship/issues/618#issuecomment-2364663325), following your draft for other sections. Please let me if you have any question or suggestion!

I am not sure whether Resource 2.06 should be titled as _... Rapid Qualitative Analysis Transcript Summary_ or _... Rapid Qualitative Analysis Transcript Summary **Template**_. 
- Technically, this is a template but in Hamilton's notes, it is titled _Transcript Summary_(p.23 in this [handout](https://www.hsrd.research.va.gov/for_researchers/cyber_seminars/archives/780-notes.pdf)). 
- In the next phase - Conduct Transcript Summaries, I assume this template will be renamed to something like _... Rapid Qualitative Analysis Transcript Summary **[P1]**_. What do you think?

**Comment by jinyan0425** [comment]
I assigned myself for tracking purposes. @pandanista

**Comment by pandanista** [comment]
@jinyan0425 
- *Rapid Qualitative Analysis Transcript Summary* sounds good to me.
- *Rapid Qualitative Analysis Transcript Summary [P1]* makes sense.
- Will finish the testing and let you know how it goes.

**Comment by pandanista** [comment]
@jinyan0425 
- I tested and edited the [template](https://github.com/hackforla/internship/issues/618#issuecomment-2327661203). See screenshot below: <img width="700" alt="Screenshot 2024-09-26 at 1 07 55 PM" src="https://github.com/user-attachments/assets/42d65109-441e-4575-9da2-1b23b375ac51">
    - I added the hand off process at the end of the template
    - I added two more action items:
        - For duplicated domains (domains with the same name), only paste them once in Spreadsheet B.
        - To keep the format consistent, when pasting the domain names into Spreadsheet B, choose Edit > Paste special > Values only. 
 Let me know what you think, and feel free to edit the template above to make more changes if needed.

-  Question:
    - One of the dependencies: RP[Replace 000]'s transcript has been transcribed, de-identified and cleaned up. [Replace with de-identification and clean-up issue numbers]
    - Should we take it off? Technically RQA doesn't need transcripts, but we are using transcripts at least for RP001's analysis.

**Comment by jinyan0425** [comment]
@pandanista 

Thanks for the meticulous edits of the template! I love the two additional action items you added, and the hand-off process.

As for your question, I would take it off for the reason you mentioned.

**Comment by pandanista** [comment]
@jinyan0425 Thank you for working on this together! 🤗
- Edited the template draft to remove the transcript dependency. 
- I think we can deploy this one now.
- Would you do the honor of deploying the [template](https://github.com/hackforla/internship/issues/618#issuecomment-2327661203) onto the repo whenever you have time? Let me know if you have any other questions.
- Once deployed onto the repo, we can close #620 and #621, and move onto the template for the second phase #624.

**Comment by jinyan0425** [comment]
@pandanista You re very welcome! Sure that I can deploy it. One quick question based on what you showed me today for the create/update research roadmap template:

> title: TWE: Research: Rapid Qualitative Analysis (RQA): Pre-process

Shall it be 

> TWE: IS[Replace YY]: RP[Replace 000]: [Replace TYPE OF PARTICIPANT] [Replace TITLE OF RESEARCH]: Rapid Qualitative Analysis Domain Names?

**Comment by pandanista** [comment]
- title: TWE: IS[Replace YY]: RP[Replace 000]: Rapid Qualitative Analysis (RQA): Pre-process

- name: Research: Rapid Qualitative Analysis (RQA) Pre-processing

- Let's use 2pt.

**Comment by jinyan0425** [comment]
@pandanista  Thanks! 

- The template has been deployed! 

- I also added the BEST PRACTICE row (row 5) to [TWE: Research: Rapid Qualitative Analysis Transcript Summary Template](https://docs.google.com/spreadsheets/d/1kHnMpIK6AHEoxFY9XhGMb5JRWPBnuDjZzahG2vovGJQ/template/preview).

- I will close #620 and #621.
